engine/render/src/render/font_renderer.cpp

```diff
@@ -294,6 +294,9 @@ namespace dmRender
 
         ro->m_ConstantBuffer = constants_buffer;
 
+        // The cache size may have changed, and we need to update the font map glyph texture
+        UpdateCacheTexture(font_map);
```

Comment:
Previously, this functionality was called each time we call add a glyph to the texture.
Now we check it before the for-loop instead.